AsyncDisplayKit/ASMapNode.mm

```diff
@@ -207,7 +207,10 @@ - (void)takeSnapshot
                     [image drawAtPoint:CGPointZero];
                     
                     // Get a standard annotation view pin. Future implementations should use a custom annotation image property.
-                    MKAnnotationView *pin = [[MKPinAnnotationView alloc] initWithAnnotation:nil reuseIdentifier:@""];
+                    __block MKAnnotationView *pin;
+                    dispatch_sync(dispatch_get_main_queue(), ^{
```

Comment:
Thanks for all the comments - I'm taking a look at fixing this right now. However, I did notice that `setAnnotations:` needs to run on the main thread at least in the case of `isLiveMap == true`, and I'm not doing that in my code since I'm setting the annotations during creation of my node. Is this intended to only be used on the main thread and missing documentation? Or should the `isLiveMap` portion be put inside of a `dispatch_async`?